channeldb/log.go

```diff
@@ -35,4 +36,5 @@ func UseLogger(logger btclog.Logger) {
 	migration12.UseLogger(logger)
 	migration13.UseLogger(logger)
 	migration16.UseLogger(logger)
+	migration18.UseLogger(logger)
```

Comment:
Will also need to be updated to be migration #22 (unless something has taken that slot itself already).

Reply:
Fixed! Changed to 23.